core/services/RestService.js

```diff
@@ -15,7 +15,27 @@ module.exports = {
       if (!ctx.params.id) {
         ctx.throw(400, "id is required")
       }
-      const instance = await yaxys.db.findOne(identity, { id: ctx.params.id })
+      let otmPopulate = []
```

Comment:
You should move this logic into the separate function into the Adapter as we discussed earlier.